server/methods.js

```diff
@@ -0,0 +1,22 @@
+ Meteor.startup(function() {
+  if(Meteor.isServer) {
+    fuzzy = Npm.require("fuzzy");
+  }
+});
+
+ Meteor.methods({
+  findDreamCollegeId: function(userInputCollege) {
+    check(userInputCollege, String);
+    var colleges = CollegesInMemory;
+    var collegeNameArray = [];
+    var collegeNameHash = {}
+    _.each(colleges, function(college) {
+      collegeNameArray.push(college.name);
+      collegeNameHash[college.name] = college._id;
+    })
+    var results = fuzzy.filter(userInputCollege, collegeNameArray);
```

Comment:
Indeed, thanks. Do you have an opinion on a fuzzy-based method or a Mongo text-search based method?  Both must happen on the server. ``fuzzy`` is a little more customizable, but requires CollegesInMemory to be present, and would need a more significant re-implementation in standalone oli.